examples/amp-story/attachment.html

```diff
@@ -65,7 +65,7 @@
           </p>
         </amp-story-grid-layer>
         <amp-story-page-attachment data-title="Lorem ipsum" layout="nodisplay">
-          <p> Lots of content. Opens to max of 80% then scrolls.</p>
+          <p>Lots of content. Opens to max of 80% then scrolls.</p>
```

Comment:
This could be removed from the PR since it's unrelated to the issue. 
Or, a comment could be added in the description to mention this typo fix.

Reply:
Done!